charts/gatekeeper/templates/gatekeeper-audit-deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         {{- toYaml .Values.auditPodAnnotations | trim | nindent 8 }}
         {{- end }}
       labels:
-{{- include "gatekeeper.podLabels" . }}
```

Comment:
Thanks for the PR! @fmorgad0 

Gatekeeper helm chart is autogenerated and these changes will be clobbered in the next release. Please refer to [Contributing to Helm Chart](https://open-policy-agent.github.io/gatekeeper/website/docs/help#contributing-to-helm-chart) for modifying the Helm chart.

Reply:
This page should not be updated yet until we cut a release.

Reply:
Thank you for showing me the documentation 🙏